pyvista/jupyter/pv_pythreejs.py

```diff
@@ -537,6 +537,17 @@ def convert_plotter(pl):
                     width += pv_ren.width + pv_ren.border_width * 2
                 grid[i, j] = convert_renderer(pv_ren)
 
+        # check for linked cameras
+        cameras = [ren.camera for ren in pl.renderers]
+        for cc, camera_a in enumerate(cameras):
+            for dd, camera_b in enumerate(cameras[cc + 1 :]):
```

Comment:
Doesn't this need to be
```suggestion
            for dd, camera_b in enumerate(cameras[cc + 1 :], start=cc + 1):
```
if we want to keep `dd` to correspond to the same kind of index as `cc`?